Dockerfile-Notebook

```diff
@@ -0,0 +1,9 @@
+FROM server:latest
+EXPOSE 8888
+
+RUN apt-get update && apt-get install -y graphviz
+RUN pip3 install jupyter
+
+COPY . /32de-python/
+
+ENTRYPOINT ["jupyter", "notebook", "--config", "deployment/jupyter_notebook_config.py"]
```

Comment:
I would at least add `tini` as this should make jupyter more stable – this is also recommended in the [Jupyter Docs](https://jupyter-notebook.readthedocs.io/en/stable/public_server.html#docker-cmd). You can also refer to the base [Jupyter Dockerfile](https://github.com/jupyter/docker-stacks/blob/master/base-notebook/Dockerfile)